src/sdk.js

```diff
@@ -173,8 +173,7 @@ class OpenForm {
   }
 
   async onLanguageChangeDone(newLanguagecode) {
-    this.formObject = await get(this.url);
-    this.render();
+    window.location.reload();
```

Comment:
unfortunately this approach is too naive - we can only trigger this in the backend project where we know what the full page looks like, but on pages (like a CMS) outside of our control where the form is embedded, we must play nice and not take over control of the whole page.

I think adding an option to accept a callback to invoke when the language change is done/requested is the best approach, so that anyone who wishes to can hook into this to do the necessary thing.